src/Square.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default props => (
+  <button className="square" onClick={props.onClick}>
```

Comment:
Similar comment here as in board... maybe the event can be named `onSelect`? This case doesn't make  such a strong case as the Board one though....

Reply:
Right, same case. I named it just like that without reading this 😄